Assets/MRTK/SDK/Experimental/RiggedHandVisualizer/RiggedHandVisualizer.cs

```diff
@@ -0,0 +1,254 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Experimental.RiggedHandVisualizer
+{
+    /// <summary>
+    /// Hand visualizer that controls a hierarchy of transforms to be used by a SkinnedMeshRenderer
+    /// Implementation is derived from LeapMotion RiggedHand and RiggedFinger and has visual parity
+    /// </summary>
+    public class RiggedHandVisualizer : MonoBehaviour, IMixedRealityControllerVisualizer, IMixedRealitySourceStateHandler, IMixedRealityHandJointHandler
+    {
+        public virtual Handedness Handedness { get; set; }
+
+        public GameObject GameObjectProxy => gameObject;
+
+        public IMixedRealityController Controller { get; set; }
+
+        public Transform Wrist;
+        public Transform Palm;
```

Comment:
I have highlighted a couple here but doc comments are useful for public properties, [mrtk coding guidelines](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#function--method-summary-headers)